src/main/java/domain/AttendanceHistory.java

```diff
@@ -1,35 +1,95 @@
 package domain;
 
+import constant.CampusOpenTime;
+import constant.Holiday;
+import java.time.LocalDate;
 import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Objects;
+import java.util.Optional;
+import util.DateFormatter;
 
 public class AttendanceHistory implements Comparable<AttendanceHistory> {
-    public final static int ABSENT_DEFAULT_HOUR = 23;
-    public final static int ABSENT_DEFAULT_MINUTE = 59;
-    private final LocalDateTime attendanceTime;
-    private final AttendanceResult attendanceResult;
-
+    private final LocalDate attendanceDate;
+    private final LocalTime attendanceTime;
+    private final AttendanceResult AttendanceResult;
```

Comment:
지금보니 오타가있군요. 

상태명 첫 글자를 소문자로 변경해주셔야할 것 같아요. 👀 

```suggestion
    private final AttendanceResult attendanceResult;
```

Reply:
앗..! 감사합니다! 수정하겠습니다.